cirq/ionq/results.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2020 The Cirq Developers
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Result types for the IonQ API."""
+
+from typing import Dict
+
+from cirq.value import digits
+from cirq import study
+
+
+def _little_endian_to_big(value: int, bit_count: int) -> int:
+    return digits.big_endian_bits_to_int(
```

Comment:
Feels like we should move this to a  `little_endian_to_big` to `cirq.value.digits`. Might be needed by others in the future.